spec/unit/controllers/internal/app_crashed_controller_spec.rb

```diff
@@ -35,13 +35,14 @@ module VCAP::CloudController
         expect(last_response.status).to eq(200)
         expect(last_response.body).to eq '{}'
 
-        app_event = Event.find(actee: diego_process.guid, actor_type: 'app')
+        app_guid = diego_process.app.guid
+        app_event = Event.find(actee: app_guid, actor_type: 'app')
 
         expect(app_event).to be
         expect(app_event.space).to eq(diego_process.space)
```

Comment:
It's only a very minor thing; as this is an **app** event, it would be more consistent to expect that the `app_event.space` equals `diego_process.app.space`. I know, it's the same, but somewhat easier to understand from my point of view.

Maybe you define a `let(:app) { diego_process.app }` and then use `app.guid` and `app.space` in the tests.

Reply:
Sounds good. Named the `let` slightly differently since `app` shadows something the controller specs use to implement `post`, `get`, etc.

https://github.com/cloudfoundry/cloud_controller_ng/commit/a90ad04913fc3fc224b83fcd6d6d23c51746078e#diff-124ca0f3421fb8907f4ce42b34e471d2ac64ede838ce5d1b16f93bfaebd31323R9